cfgov/unprocessed/js/modules/external-site-redirect.js

```diff
@@ -6,29 +6,39 @@
    ========================================================================== */
 
 'use strict';
-var $ = require( 'jquery' );
 
 /**
  * Set up event handler for links and determine if link is external or not.
  */
 function init() {
-  $( '#main, footer' ).on( 'click', 'a', function( event ) {
+  var elements = document.querySelectorAll( '#main, body > footer' );
 
-    var url = this.href;
-    // Regex to determine if link URL is external.
-    // Futher explanation can be viewed
-    // at https://regex101.com/r/xT7sL5/2.
-    var externalURLArray =
-        ( /(https?:\/\/(?:www\.)?(?!.*gov)(?!(content\.)?localhost).*)/g )
-        .exec( url );
+  for ( var i = 0; i < elements.length; ++i ) {
+    elements[i].addEventListener( 'click', _handleClick, false );
+  }
+}
+
+/**
+ * Handle the element click event.
+ * @param {ClickEvent} event A click event.
+ */
+function _handleClick( event ) {
+  var element = event.target;
+  if ( element && element.tagName !== 'A' ) element = element.parentNode;
+  if ( element === null || element.tagName !== 'A' ) return;
```

Comment:
So the problem is that certain anchor elements contain spans. The event.target will be a span in that scenario.
`<a><span>text</span>`